crates/nao/src/lib.rs

```diff
@@ -34,10 +34,15 @@ impl Nao {
         host: Ipv4Addr,
         timeout_seconds: u32,
     ) -> Result<Self> {
+        #[cfg(target_os = "macos")]
+        let timeout_flag = "-t";
+        #[cfg(not(target_os = "macos"))]
+        let timeout_flag = "-w";
```

Comment:
These could be constants, right?

```suggestion
        #[cfg(target_os = "macos")]
        const TIMEOUT_FLAG: &str = "-t";
        #[cfg(not(target_os = "macos"))]
        const TIMEOUT_FLAG: &str = "-w";
```

Reply:
Yes they could be. I made the change in the latest commit to this branch.